src/cloudai/systems/slurm/strategy/slurm_command_gen_strategy.py

```diff
@@ -173,8 +169,9 @@ def _write_sbatch_script(self, args: Dict[str, Any], env_vars_str: str, srun_com
             "#!/bin/bash",
             f"#SBATCH --job-name={args['job_name']}",
             f"#SBATCH -N {args['num_nodes']}",
+            f"#SBATCH --gres=gpu:8"
```

Comment:
I am not sure whether it is okay to hard-code the number of GPUs per node. What if another cluster has 4 GPUs per node instead of 8? Could we come up with a more general approach?